java/vector/src/main/codegen/templates/FixedValueVectors.java

```diff
@@ -162,7 +166,7 @@ public void allocateNew(final int valueCount) {
 
   @Override
   public void reset() {
-    allocationSizeInBytes = INITIAL_VALUE_ALLOCATION;
+    allocationSizeInBytes = INITIAL_VALUE_ALLOCATION * ${type.width};
```

Comment:
I'm just wondering if maybe this was correct before and it was wrong in initialization by multiplying the `type.width`?  `INITIAL_VALUE_ALLOCATION` is 4096 which seems more like a number of bytes than a value count.  I guess it depends on what makes more sense for different vector types: 1) to allocate a constant number of bytes for each type or 2) allocate space for a constant number of values for that type ?